docs/dxp/7.x/en/developing-applications/reference/portlet-descriptor-to-osgi-service-property-map.md

```diff
@@ -81,6 +81,7 @@ The standard portlet descriptor mappings are first.
 |`/liferay-portlet-app/portlet/active`|`com.liferay.portlet.active=<boolean>`|
 |`/liferay-portlet-app/portlet/add-default-resource`|`com.liferay.portlet.add-default-resource=<boolean>`|
 |`/liferay-portlet-app/portlet/ajaxable`|`com.liferay.portlet.ajaxable=<boolean>`|
+|`/liferay-portlet-app/portlet/application-type`|`com.liferay.portlet.application-type=full-page-application` and/or `com.liferay.portlet.application-type=widget`|
```

Comment:
@jhinkey For actual example of the usage see https://github.com/liferay/liferay-portal/blob/b75cf72ed573622f5c2fff167a5c8b58904414ca/modules/apps/blogs/blogs-web/src/main/java/com/liferay/blogs/web/internal/portlet/BlogsPortlet.java#L41, this option only supports 2 values `full-page-application` and `widget` but one can declare both as shown in `BlogsPortlet`.